app/Models/Album.php

```diff
@@ -27,6 +27,7 @@
  * @property string|null       $parent_id
  * @property Album|null        $parent
  * @property Collection<Album> $children
+ * @property int               $num_subalbums    The number of children.
```

Comment:
I would use `$num_children` to be consistent with the `$children` attribute.

Reply:
The key `children` gets later renamed to `albums` (l.186) that's why I went with albums to begin with. However, I like your emphasis on consistency and now followed your suggestion. However, I then also renamed the key but to 'num_subalbums`. I have no strong preferences either way but using `num_albums` I find a bit confusing because it is not immediately clear whether it refers to this album (the parent, i.e. its siblings) or its children.